examples/templates/README.md

```diff
@@ -30,14 +54,22 @@ In order to create new code examples with these templates, each new example must
     "url": "https://github.com/handsontable/handsontable/issues"
   },
   "author": "Handsoncode <hello@handsontable.com>",
-  "license": "MIT",
+  "license": "SEE LICENSE IN LICENSE.txt",
   "dependencies": {
     "handsontable": "latest",
-    "@handsontable/{{ WRAPPER NAME }}": "latest" // only for frameworks
+    "@handsontable/{{ WRAPPER NAME }}": "latest" // only for frameworks, e.g. "@handsontable/react": "latest"
   }
 }
 ```
 
-#### Don't touch
+In `package.json` are also scripts ***you should never modify or remove***, because those scripts are used by our CI/CD pipeline : `npm run start`, `npm run build` and `npm run test`.
+
+The only thing you have to modify is `"name"`, `"description"` and `"@handsontable/{{ WRAPPER NAME }}"` (if you are creating a code example made in a framework like React, Angular, or Vue). And, of course, you can install dependencies.
+
+#### 3. Develop
+
+After the dependencies are installed, run npm run start that builds the project and watches for any saved code changes. The command opens the project on port 8080 so you can open the example in your browser at `localhost:8080`.
+
+#### 4. Publishing
 
-Each example have its own scripts ***you should never modify or remove***, because those scripts are used by our CI/CD pipeline : `npm run start`, `npm run build` and `npm run test`.
+After you're done, `git add examples/next`, `git commit` and `git push` your new code examples to the Handsontable repo. Open a new PR and wait for the review!
```

Comment:
Now you have written your code, it is ready to be pushed. Follow the steps below to make it possible:

1. `git add examples/next`     @Osky772 I'm not about it. Will it be always examples/next folder?
2. `git commit`
3. `git push`

After completing these steps you can open a PR on GitHub, and assign a reviewer.

Reply:
Hmm, not always. It makes sense to create a new example, for a client, using the `8.0.0` version.